File: prob-10/prob10.c
#include <stdio.h>

main()
{
	int num, i;
	long long int sum;

	num = 2;
	i = 2;
	sum = 0;

	while(num < 2000000){
		if(num % i == 0.0){
			if(num == i){
				sum += num;
			}
			i = 2;
			num++;
		}
		else{
			i++;
		}
	}
	printf("%lld\n", sum);
}

File: prob-10/prob10.py
def isPrime(num):
    """Checks if the given number is a prime"""
    isPrime = True
    for i in range(2, num // 2):
        if num % i == 0:
            isPrime = False
            break
    return isPrime

primes = []

for i in range(2, 2 * 10 ** 6):
    if isPrime(i):
        primes.append(i)

print sum(primes)
